src/seed.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-console */
+
+import { createConnection, getConnectionOptions, getConnection, ConnectionOptions } from 'typeorm';
+import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
+import path from 'path';
+import { Team } from './entities/team';
+import { Judge } from './entities/judge';
+import { env } from './env';
+import logger from './logger';
+
+async function initDatabase(): Promise<void> {
```

Comment:
You should be able to just import and call the same initDatabase that the app does so you don't need to write your own

Reply:
I think don't the code exports `initDatabase` anywhere. I just copied this function from `app.ts`. I guess we can export `initDatabse` in `app.ts` and import it here, let me know what you think.

Reply:
Gotcha, yeah let's export that and use it here 👍